.circleci/circle-ci.sh

```diff
@@ -33,24 +33,32 @@ JTR=../run/john
 # Build and testing
 if [[ $1 == "BUILD" ]]; then
 
+    if [[ "$PLUGS" == "none" ]]; then
+        rm -f *plug.[ch]
+    fi
+
     if [[ -n "$WINE" ]]; then
         do_Copy_Dlls
         export WINEDEBUG=-all
     fi
 
     # Configure and build
     if [[ "$TARGET_ARCH" == "i686" ]]; then
-        ./configure --host=i686-w64-mingw32 --build=i686-redhat-linux-gnu --target=i686-w64-mingw32
+        ./configure ${CONF_FLAGS} --host=i686-w64-mingw32 --build=i686-redhat-linux-gnu --target=i686-w64-mingw32
```

Comment:
Where does CONF_FLAGS come from?

Reply:
Good catch, is that still there? I added it, then realized I could use the existing `BUILD_OPTS` instead.
I'll drop `CONF_FLAGS` and add `BUILD_OPTS` instead (not currently used for these lines but may come handy).